xbmc/pvr/PVRTypes.h

```diff
@@ -59,5 +59,14 @@ namespace PVR
   class CPVRGUIActions;
   typedef std::shared_ptr<CPVRGUIActions> CPVRGUIActionsPtr;
 
+  class CPVREpgContainer;
+  typedef std::shared_ptr<CPVREpgContainer> CPVREpgContainerPtr;
```

Comment:
why is this a shared pointer? imo epgContainer could just be a plain member of PVRManager and clients could ask for a reference. it has the same lifetime as the pvrManager object.